src/session/index.ts

```diff
@@ -1,7 +1,8 @@
 //#region > Imports
 //> Cookie Utils
```

Comment:
Please remove legacy comment.

Reply:
Yes, the comment is legacy but I see no reason the remove it. I will replace it with one that fits.
In the upcoming documentation update all imports of the client are refined to fit the seek-tonic anyways.

Ref: https://github.com/snek-at/tonic